dockerfiles/stopall.sh

```diff
@@ -34,11 +34,11 @@ while [ "$#" -gt 0 ]; do
          CLUSTERNAME=$1
          ;;
       -e) UNINSTALL_ELK=0
-	     echo "elastic4hpcclogs will not be stopped..."
+       echo "elastic4hpcclogs will not be stopped..."
```

Comment:
Minor and unrelated to target Jira, but ultimately will not confuse anybody and resolves minor annoyance without spending unnecessary developer time fixing it in dedicated jira.